src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -710,6 +711,14 @@ public void deleteFile(TestConfiguration.User user, UUID datasetId, String fileI
     return dataRepoClient.drsGet(user, "/ga4gh/drs/v1/objects/" + drsObjectId, DRSObject.class);
   }
 
+  public DrsResponse<bio.terra.model.DRSAccessURL> getObjectAccessUrl(
```

Comment:
You've got a fully qualified class name...could be a followon but maybe convert this to the not full qualified version